### ISSUE TITLE
`<p>` tags wrapping the inputs...

### ISSUE THREAD

**Issue by haydnba** [fac-13/ab-fac#33]
https://github.com/fac-13/ab-fac/blob/051cefe6edbe43a65115af9c2158704a10880a5b/index.html#L166

I **think** that wrapping the inputs in paragraph tags leads to issues being raised by accessibility audits - although this is what formspree does by default it seems...

Also, there are no labels for the form inputs which screen readers don't like

🐿

**Comment by helenzhou6** [comment]
For lines of code https://github.com/fac-13/ab-fac/blob/master/index.html#L137-L146

Just a comment re:adding labels on your contact form, you can actually put your ```input``` tags inside a ```label``` tag (that may make more sense than using ```p``` tags). E.g.:

```
                <label for="form-name">First Name
                    <input name="First Name" id="form-name" type="text" aria-label="Name" 
 placeholder="Your Name" required/>
                </label>
```
I know this means you will have 'First Name' displayed so you'll need another class to hide it (like ```<span class="is-hidden">First Name</span>```)? 🤔

**Comment by eadehemingway** [comment]
thanks for this, 
we have just altered it to use 'arialabelledby' which now works with chromevox!
